snuba/datasets/schemas/__init__.py

```diff
@@ -0,0 +1,68 @@
+from abc import ABC, abstractmethod
+from typing import Mapping, List
+
+from snuba import settings
+from snuba.clickhouse.columns import ColumnSet
+
+
+def local_dataset_mode():
+    return settings.DATASET_MODE == "local"
```

Comment:
I'm not sure that this belongs here but I'm also not sure where it *does* belong. Maybe somewhere in util?